tool/grind.dart

```diff
@@ -251,10 +251,7 @@ dart run protoc_plugin "\$@"
     run('chmod', arguments: ['a+x', 'build/protoc-gen-dart']);
   }
 
-  if (Platform.environment['UPDATE_SASS_PROTOCOL'] != 'false') {
-    cloneOrCheckout("https://github.com/sass/sass.git", "main",
-        name: 'language');
-  }
+  _updateLanguageRepo();
```

Comment:
The function returns a value, but it's not being used. Is that intended?

Reply:
Oops, missed a usage. Good catch!